app/editor/src/features/content/form/styled/ContentForm.tsx

```diff
@@ -8,10 +8,6 @@ export const ContentForm = styled.div`
     padding: 0;
   }
 
-  #txa-summary {
```

Comment:
Limiting size of textarea, also put the styling closer to the element in question.  Having it here only makes sense if the child components are not going to have styling.